### ISSUE TITLE
Find out best learning rate for training

### ISSUE THREAD

**Issue by carsten-wenderdel** [carsten-wenderdel/wildbg#15]
Depending on the size of the neural network, the inputs and the amount of training data different learning rates lead to the best neural network. It would be great, if we don't have manually try that out but if somehow this could be figured out automatically.
